pkg/test/utils/kubernetes.go

```diff
@@ -942,7 +942,7 @@ func RunCommands(logger Logger, namespace string, command string, commands []kud
 		stdout := &bytes.Buffer{}
 		stderr := &bytes.Buffer{}
 
-		logger.Log("Running command:", cmd)
+		logger.Logf("Running command: %s %s", command, cmd)
```

Comment:
The handling of `command` in `RunCommands` is weird, most of the time it's an empty string. I'd rather have us refactor these functions. But this isn't the scope of this PR, and as this improves logging for the case that `command == "kubectl"`, I'm fine with this addition.